src/mark.ts

```diff
@@ -119,12 +120,19 @@ export interface MarkConfig extends VgMarkConfig {
   /**
    * Whether the mark's color should be used as fill color instead of stroke color.
    * All marks except "point", "line", and "rule" are filled by default.
+   *
+   * __Default value:__ `true` for all marks except `point` and `false` for `point`.
+   *
+   * __Applicable for:__ `bar`, `point`, `circle`, `square`, and `area` marks.
+   *
    */
   filled?: boolean;
 
   // TODO: remove this once we correctly integrate theme
   /**
    * Default color.
+   *
+   * __Default value:__ <span style="color: #4682b4;">&#9632;</span> blue (`""#4682b4"`)
```

Comment:
>  blue (`""#4682b4"`)

Just `"#4682b4"` is better.  "blue" is a valid color value so it might be a little confusing.

(Make sure to remove the typo that include one extra quote.